dune-project

```
@@ -0,0 +1,3 @@
+(lang dune 1.7)
+(name nocrypto)
+(version %%VERSION_NUM%%)
```

I don't think that this is necessary

```suggestion
(version %%VERSION_NUM%%)
```

---

Correct, it's not strictly necessary. But it adds version to `META` files.

---

META files are automatically generated with the correct version. For example, ̀cstruct` doesn't have `(version)` here, and you can check that the installed META file has the correct version.

---

This is not what I'm seeing while playing with a pinned package. I won't pretend to understand why.